### PR TITLE
Resolves all flake8 issues, including F401 and F403

### PR DESCRIPTION
**Pull request by roman-verchikov**
Following flake8 errors were ignored previously:
 - F401 - module imported but unused
 - F403 - ‘from module import *’ used; unable to detect undefined names

This patch resolves all these issues.

### CODE REVIEW

File: cloudferrylib/utils/__init__.py [hunk 1]
```diff
@@ -11,7 +11,3 @@
 # implied.
 # See the License for the specific language governing permissions and#
 # limitations under the License.
```
Comment (miarmak) [inline cloudferrylib/utils/__init__.py]: I guess we can remove this copyright as well.

**Comment by miarmak** [comment]
Looks good overall, except of 1 small comment inline.

File: cloudferrylib/utils/__init__.py (hunk 1 above)
Comment (roman-verchikov) [inline cloudferrylib/utils/__init__.py]: Well this seems to be unrelated to the purpose of the patch, plus it's not 100% clear whether this particular file needs to be copyrighted or not, according to [Apache FAQ](http://www.apache.org/legal/src-headers.html#faq-exceptions):

> Q: What files in an Apache release do not require a license header?
> A: A file without any degree of creativity in either its literal elements or its structure is not protected by copyright law; therefore, such a file does not require a license header. **If in doubt about the extent of the file's creativity, add the license header to the file**.

Since we're in doubt, let's keep the header there :)

**Comment by roman-verchikov** [comment]
Can be merged
